docs/excel/excel-data-types-add-properties-to-basic-cell-values.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Add properties to basic cell values
+description: Add properties to basic cell values.
+ms.topic: how-to #Required; leave this attribute/value as-is
+ms.date: 04/14/2025
+ms.localizationpriority: medium
+---
+
+# Add properties to basic cell values (preview)
+
+Add properties to basic cell values in Excel to associate additional information with the values. Similar to entity values, you can add properties to the **string**, **double**, and **Boolean** basic types. Each property is a key/value pair. The following example shows the number 104.67 (a double) that represents a bill with added fields named **Drinks**, **Food**, **Tax**, and **Tip**.
+
+:::image type="content" source="../images/data-type-basic-fields.png" alt-text="Screen shot of the drinks, food, tax, and tip fields shown for the selected cell value.":::
```

Comment:
Note that screenshot is one word, if you opt to keep the phrase "Screenshot of..."

```suggestion
:::image type="content" source="../images/data-type-basic-fields.png" alt-text="Screenshot of the drinks, food, tax, and tip fields shown for the selected cell value.":::
```